src/App.jsx

```diff
@@ -13,34 +15,96 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
As a recommendation: 
If you're familiar with `enum` it'd be nice to use that one here

Reply:
Thanks!)
I didn't know about this feature, I guess it is used in TypeScript.